### PR TITLE
case study

### PR DESCRIPTION
**Pull request by huskerman34**
Didnt test  brackets wasnt working right

### CODE REVIEW

**Comment by huskerman34** [comment]
chapter 12 added into this case study and also fixed the errors that you reported.  I still was having issues  with the images fitting the pages in all three pages. Could you look into this please!!!

File: chapter 8 casestudy/activities.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<!Doctype html>
```
Comment (mbMosman) [inline chapter 8 casestudy/activities.html]: Only the files in the root folder will be graded.

File: pacifictrails.css [hunk 2]
```diff
@@ -0,0 +1,361 @@
+                        body {
```
Comment (mbMosman) [inline pacifictrails.css]: Font colors and styles are not correct.

File: pacifictrails.css [hunk 3]
```diff
@@ -0,0 +1,361 @@
...
+            background-color: #4f6d93;
+            background-image: linear-gradient(to bottom, #ffffff, #4f6d93);
+            background-repeat: no-repeat;
+            margin: 0;
+        }
+        
+        nav {
```
Comment (mbMosman) [inline pacifictrails.css]: Font styles are not correct.

File: pacifictrails.css [hunk 4]
```diff
@@ -0,0 +1,361 @@
...
+            padding: 20px;
+        }
+        
+        h1 {
+            margin-top: 0;
+        }
+        
+        h2 {
+            color: #3399CC;
+            font-family: Georgia;
```
Comment (mbMosman) [inline pacifictrails.css]: Still not using full font-stack for the font-family.

File: pacifictrails.css [hunk 5]
```diff
@@ -0,0 +1,361 @@
...
+        .resort {
+            color: #5C7FA3;
+        }
+        
+        a {
+            text-align: center;
+        }
+        
+        h3 {
+            font-family: sans-serif;
```
Comment (mbMosman) [inline pacifictrails.css]: Again, not using full font-stack for the font family.

File: pacifictrails.css [hunk 6]
```diff
@@ -0,0 +1,361 @@
...
+        th {
+            padding: 5px;
+            border: 1px solid #3399CC;
+        }
+        
+        td {
+            text-align: center;
+        }
+        
+        text {
```
Comment (mbMosman) [inline pacifictrails.css]: This is not an HTML tag, so it should begin with a . for a style class or a # for an id.

File: pacifictrails.css [hunk 7]
```diff
@@ -0,0 +1,361 @@
...
+        #gallery a {
+            text-decoration: none;
+            font-style: italic;
+            color: #eaeaea;
+        }
+        
+        @media only screen and (max-width:37.5em) {
+            #gallery {
+                display: none;
+            }
```
Comment (mbMosman) [inline pacifictrails.css]: The video should also be hidden for mobile displays.

File: activities.html [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+            <p>While anytime is a good time for bird watching at Pacific Trails, we offer good time for bird watching trips at sunrise several times a week.</p>
+            <div id="gallery">
+                <figure>
+                    <img src="images/photo6.jpg" width="200" height="150" alt="Sunset on The Coast"></figure>
+
+                <ul>
+                    <li>
+                        <a href="images/photo2.jpg"><img src="images/photo2thumb.jpg" width="100" height="75" alt="Images">
+                            <span><img src="images/photo2.jpg" width="200" height="150" alt="images"><br>Images </span></a>
+                        <li>
```
Comment (mbMosman) [inline activities.html]: Closing li tags are missing and captions are incorrect.

File: pacifictrails.css [hunk 9]
```diff
@@ -0,0 +1,361 @@
...
+        
+        textarea {
+            display: block;
+            margin-bottom: 20px;
+        }
+        
+        input[type="submit"] {
+            margin-left: 130px;
+        }
+
```
Comment (mbMosman) [inline pacifictrails.css]: The gallery CSS should be outside of and before your media queries.

File: pacifictrails.css [hunk 10]
```diff
@@ -0,0 +1,361 @@
...
+        }
+        
+        video,
+        embed {
+            float: right;
+            padding-left: 20px;
+        }
+        
+        #gallery {
+            height: 200px;
```
Comment (mbMosman) [inline pacifictrails.css]: needs to specify position is relative.

File: pacifictrails.css [hunk 11]
```diff
@@ -0,0 +1,361 @@
...
+            position: absolute;
+            top: 16px;
+            left: 320px;
+            text-align: center;
+            opacity: 1;
+        }
+        
+        figure {
+            position: absolute;
+            right: 280px;
```
Comment (mbMosman) [inline pacifictrails.css]: This should be left, not right.

File: pacifictrails.css [hunk 12]
```diff
@@ -0,0 +1,361 @@
...
+            text-align: center;
```
Comment (mbMosman) [inline pacifictrails.css]: should also specify opacity.

File: reservation.html [hunk 13]
```diff
@@ -0,0 +1,68 @@
...
+                <textarea name="myComments" id="myComments" rows="2" cols="30" required="required" placeholder="your comments here"></textarea>
+                <input type="submit" value="Submit">
+                <br>
+                <br>
+            </form>
+            <div id="contact">
+                Pacific Trails Resort
+                <br> 12010 Pacific Trails Road
+                <br> Zephyr, CA95555
+                <br> 888-555-5555
```
Comment (mbMosman) [inline reservation.html]: This should be a link for mobile.

**Pull request closed without merging**